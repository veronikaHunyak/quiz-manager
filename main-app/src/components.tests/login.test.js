import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import Login from "../components/login";

describe("Login", () => {
  it("should render correctly", () => {
    expect(
      toJson(shallow(<Login loginStateChange={jest.fn()} />))
    ).toMatchSnapshot();
  });
});
