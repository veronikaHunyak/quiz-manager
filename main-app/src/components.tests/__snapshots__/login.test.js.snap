// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render correctly 1`] = `
<div
  className="container"
>
  <form
    action=""
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <label
        for="username"
      >
        Username:
      </label>
      <input
        className="form-control"
        id="username"
        name="username"
        onChange={[Function]}
        type="text"
      />
    </div>
    <div
      class="form-group"
    >
      <label
        for="password"
      >
        Password:
      </label>
      <input
        className="form-control"
        id="password"
        name="password"
        onChange={[Function]}
        type="password"
      />
    </div>
    <button
      class="btn btn-default"
      type="submit"
    >
      Login
    </button>
  </form>
</div>
`;
